Adyen/Model/PrefilledShopperInformation.swift

```diff
@@ -61,13 +61,13 @@ extension PrefilledShopperInformation {
     /// A structure that defines the basic properties for the shopper's card.
     public struct CardInformation {
 
-        /// The card's holdername.
-        public let holdername: String
+        /// The card's holder name.
+        public let holderName: String
```

Comment:
This is breaking change. 

Also, don't think we should prefill Card Holder name 🤔

Reply:
yes, it is, but I really don't want to deprecate it or make an exception for it, what do you think?